python/ray/scripts/scripts.py

```diff
@@ -47,6 +50,29 @@ def check_no_existing_redis_clients(node_ip_address, redis_client):
             raise Exception("This Redis instance is already connected to "
                             "clients with this IP address.")
 
+logging_options = [
+    click.option(
+        "--log-old-style/--log-new-style",
+        is_flag=True,
+        default=True,
+        help=("Use old logging.")),
```

Comment:
let's elaborate more on this - you should mention what are the differences and what should users be looking at in the code. 

Also, if future users want to modify existing logging messages, how should they go about it?